models/wrf/model_mod.f90

```diff
@@ -697,7 +697,7 @@ subroutine static_init_model()
    wrf%dom(id)%type_u      = get_type_ind_from_type_string(id,'U')
    wrf%dom(id)%type_v      = get_type_ind_from_type_string(id,'V')
    wrf%dom(id)%type_w      = get_type_ind_from_type_string(id,'W')
-   wrf%dom(id)%type_t      = get_type_ind_from_type_string(id,'T')
+   wrf%dom(id)%type_t      = get_type_ind_from_type_string(id,'THM')
```

Comment:
I can't comment on the line in the review, but the default_table should be updated to 'THM       '  rather than 'T     ' on ~line7468

https://github.com/NCAR/DART/blob/f8325c86412f43b049f8b47f6b611f74d005bf0c/models/wrf/model_mod.f90#L7468

Reply:
Completed